lib/web_console/request.rb

```diff
@@ -32,6 +32,18 @@ def acceptable_content_type?
       content_type.blank? || content_type.in?(acceptable_content_types)
     end
 
+    # Returns whether the request is acceptable
+    #
+    # It returns true if required version and current version are same.
+    # And it also returns true if a request includes no vendor media type.
+    def acceptable_version?
+      return true unless accepts.any? do |mime|
```

Comment:
> I think we should move this to the Request class file, so everything it depends on is defined in that file.

@gsamokovarov Exactly, I will move it :)

> Why do we wanna do that? Won't we always return true this way? We can always match our JavaScripts as it is now to send the version.

Oh, I see. I was thinking of the request sent by the current Web Console, but yeah I think now it is unnecessary to support.

> By the way, mime instances respond to ==, so you can always compare them with the autogenerated constant like mime == Mime::WEB_CONSOLE.

I get it.

Thanks.